Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -644,7 +633,7 @@ struct ActivityStreamTracker {
             "build": AppInfo.buildNumber,
             "locale": Locale.current.identifier,
             "release_channel": AppConstants.BuildChannel.rawValue
-            ], validate: true)
+            ] as [String: Any], validate: true)
```

Comment:
Type checking can take a while when the type of an array is not specified. https://thatthinginswift.com/debug-long-compile-times-swift/

This is actually a pretty big problem in our sql queries some of which take upwards of 500ms to compile. Swift 3 was suppose to fix this but doesnt look like much changed.